src/components/Popup/index.module.scss

```
@@ -0,0 +1,124 @@
+/* src/components/Popup/index.module.scss */
+
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 9999;
```

with tailwindcss this will be:
```suggestion
  @apply fixed top-0 left-0 w-full h-full flex justify-center items-center;
  background-color: rgba(0, 0, 0, 0.5);
  z-index: 9999;
```